test/integration/deprecated_TEST.cc

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+#include <ignition/gui/Plugin.hh>
+
+/////////////////////////////////////////////////
+// Make sure the ignition namespace still works
+TEST(Deprecated, IgnitionNamespace)
+{
+  ignition::gui::Plugin plugin;
```

Comment:
Can we test a method too? Eg.,
```c++
EXPECT_TRUE(plugin.Title().empty())
```

Reply:
https://github.com/gazebosim/gz-gui/compare/d0efc89b7a7b14b684b01aae1e59a9234f3e423b..180f036b24f4a3af584d819864063c87bb1b0733